### DISCUSSION TITLE
Enable continuous page view

### DISCUSSION THREAD

**Discussion by aamodio** [RussCoder/djvujs#60] [Q&A] [answered]
Hi,

Can't find the way to enable continuous page view as in your example https://djvu.js.org. In my installation the button there isn't.

Thanks,
A.

**Comment by RussCoder** [comment, marked as answer]
It's because your documents are indirect (multi-file) DjVus. For such documents the continuous scroll mode isn't available.  
I made it so, because in this case the size of each page isn't known in advance, so I cannot count the total height of the document and correspondingly you will not be able to scroll the document freely as it is in case of any a single-file djvu. 

However, I have already thought about it, and maybe I can just use the size of the first page as a default. Of course, when the pages are loaded, their sizes can differ, so the scrollbar will move, but it's not a big issue.

**Reply by aamodio** [reply]
Thanks @RussCoder confirm, it is an Indirect file.
